djangosaml2/views.py

```diff
@@ -138,10 +138,17 @@ def login(request,
 
     kwargs = {}
     # pysaml needs a string otherwise: "cannot serialize True (type bool)"
-    if getattr(conf, '_sp_force_authn'):
-        kwargs['force_authn'] = "true"
-    if getattr(conf, '_sp_allow_create', "false"):
-        kwargs['allow_create'] = "true"
+    if hasattr(conf, '_sp_force_authn'):
+        if getattr(conf, '_sp_force_authn') in [0, False, 'false']:
```

Comment:
This is a bit pedantic, but it is better to use direct attribute access if you know the attribute exists, i.e. `conf._sp_force_authn`. If you examine the bytecodes for `getattr`, it does a _bit_ more behind the scenes compared to a normal attribute access. But the performance difference is largely negligible which is why this comment is more pedantic than anything.

Reply:
what would you adopt as a definitive approach?

Reply:
I would go with `conf._sp_force_authn` in this case since the previous line already confirms that the attribute is available.

Reply:
done

Reply:
you're right, done:
https://github.com/knaperek/djangosaml2/commit/c1b568b01f769b23f4d8eaeb0e550583c22562a9